media_driver/cmake/media_utils.cmake

```diff
@@ -59,5 +60,15 @@ macro (MediaAddCommonTargetDefines target)
     endif()
 endmacro()
 
+# find external libs, if not found, print an error message and abort cmake
+macro (find_external_libs EXTERNAL_LIBS)
```

Comment:
I think this macro is not needed. plkg_check_modules  will error out by itself if you call it with REQUIRED like this:

```
--   No package 'XYZ' found
CMake Error at /usr/share/cmake3/Modules/FindPkgConfig.cmake:418 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake3/Modules/FindPkgConfig.cmake:585 (_pkg_check_modules_internal)
  CMakeLists.txt:484 (pkg_check_modules)
```

I.e. this macro gives useless over complexity and redefinies already written function (pkg_check_modules)

Reply:
Thanks for your reminder, this macro is useless, I have removed it.